extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -8,14 +8,13 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 	'ext.wikia.adEngine.adLogicPageViewCounter',
 	require.optional('wikia.abTest'),
 	require.optional('ext.wikia.adEngine.lookupServices'),
-	require.optional('ext.wikia.adEngine.krux')
+	require.optional('ext.wikia.krux')
```

Comment:
Let' name the Krux module wikia.krux and put it along the other wikia.* modules.